spec/helpers/searches_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
```

Comment:
I would also remove this helper spec, but instead I would write a feature test that tests the search functionality!